src/components/Select/Select.scss

```diff
@@ -148,6 +148,10 @@ $stacking-order: (
 
 .newDesignLanguage {
   .Backdrop {
+    // 'position' needs to sit below focus-ring since it will be overwritten
+    // with relative when the focus ring style is 'base'
+    // stylelint-disable-next-line order/properties-order
+    position: absolute;
```

Comment:
`yarn format` keeps making this change, which breaks the focus ring. You'll need to drop this change.

Reply:
I'll fix this up in a sec